### PR TITLE
Restrict tags to lowercase, add description and name length limits

### LINKED ISSUE #182: Restrict tags to lowercase only

### PR DESCRIPTION
**Pull request by noelmathewisaac**
Closes #182

### CODE REVIEW

File: src/main/java/seedu/taskify/logic/commands/AddCommand.java [hunk 1]
```diff
@@ -31,6 +31,7 @@
 
     public static final String MESSAGE_SUCCESS = "New task added: \n%1$s";
     public static final String MESSAGE_DUPLICATE_TASK = "This task already exists in Taskify";
+    public static final String MESSAGE_LONG_NAME = "This task already exists in Taskify";
```
Comment (khiaxeng) [inline src/main/java/seedu/taskify/logic/commands/AddCommand.java]: Did you mean to change this String?

Comment (noelmathewisaac) [reply src/main/java/seedu/taskify/logic/commands/AddCommand.java]: Forgot to remove this line. Updated it.

**Review by khiaxeng — APPROVED** [review]
LGTM

**Review by hengyongming — APPROVED** [review]
LGTM

**Pull request merged**